roles/sushy_emulator/tasks/collect_details.yml

```diff
@@ -69,20 +69,30 @@
   when:
     - cifmw_libvirt_manager_uuids is not defined
   block:
+    - name: "Ensure file exists: {{ _uuid_file }}"
+      register: _uuid_file_status
+      ansible.builtin.stat:
+        path: "{{ _uuid_file }}"
+
     - name: "Slurp content of: {{ _uuid_file }}"
-      ansible.builtin.slurp:
-        src: "{{ _uuid_file }}"
-      register: _libvirt_uuids_file
+      when: _uuid_file_status.stat.exists
+      block:
+        - name: "Slurp content of: {{ _uuid_file }}"
+          ansible.builtin.slurp:
+            src: "{{ _uuid_file }}"
+            register: _libvirt_uuids_file
```

Comment:
This register is wrongly indented.

Reply:
Interesting the linter didn't catch this :(

Reply:
Done

Reply:
haha, I missed that as well 😅